charts/ai-software-templates/chatbot/0.1.0/templates/tests/test-connection-app.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-test"
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  containers:
+    - name: wget
+      image: quay.io/quay/busybox:latest
+      command: ["wget"]
+      args: ["{{ .Release.Name }}:11222/console"]
```

Comment:
What endpoint is just job supposed to ping? 

This job seems like it's pinging the chatbot service, but I don't think the chatbot app exposes port 11222 or a console endpoint.

Reply:
@johnmcollier seems like something got messed up with my pod here. I've pushed a fixed version. I'm using the application port and release name to ping.